lib/editor/components/CreateSnapshotModal.js

```diff
@@ -54,6 +58,12 @@ class CreateSnapshotModal extends Component<Props, State> {
   messages = getComponentMessages('CreateSnapshotModal')
 
   _onTogglePublish = (e: SyntheticInputEvent<HTMLInputElement>) => {
+    // If unchecking publishNewVersion, we need to also uncheck publishProprietaryFiles
+    if (!e.target.checked) this.setState({publishProprietaryFiles: false})
+    this.setState({[e.target.name]: e.target.checked})
```

Comment:
```suggestion
    this.setState({[e.target.name]: !e.target.checked})
```
This can probably be cleaned up into this right

Reply:
Well, we still need to uncheck the publishProprietaryFiles checkbox if the general publishing one is unchecked. So we still need the line above it.

Reply:
thanks for clarifyihng